src/components/Slider/Slider.spec.tsx

```diff
@@ -0,0 +1,36 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { mount } from "@cypress/react";
+import { useState } from "react";
+import Slider from "./Slider";
+import css from "./Slider.module.css";
+
+const SLIDER_ITEMS = [
+    { id: "a", name: "abc" },
+    { id: "b", name: "def" },
+    { id: "c", name: "ghi" },
+];
+
+describe("Slider Component", () => {
+    it("should render foo text correctly", () => {
+        const Component = () => {
+            const [active, setActive] = useState(SLIDER_ITEMS[0].id);
+
+            return <Slider items={SLIDER_ITEMS} activeItemId={active} onChange={setActive} />;
+        };
+
+        mount(<Component />);
```

Comment:
```suggestion
        const onChangeStub = cy.stub().as("onChangeStub");

        mount(<Slider items={SLIDER_ITEMS} activeItemId={SLIDER_ITEMS[0].id} onChange={onChangeStub} />);
```

We can use cypress stub here

Reply:
I'm not sure if a stub would work for the state change. I have updated the tests, would be great if you could have another look at it :)